k8s_itests/Makefile

```diff
@@ -0,0 +1,16 @@
+# You need kind installed to run this!
+
+CLUSTER  := paasta-k8s-test-cluster
+export KIND_CLUSTER=$(CLUSTER)
+
+all: clean init itest
+
+init:
+	kind create cluster --name=$(KIND_CLUSTER) --config=./deployments/cluster.yaml
+
+itest:
+	export KUBECONFIG=`kind get kubeconfig-path --name="paasta-k8s-test-cluster"`; ../.paasta/bin/tox -e k8s_itests
```

Comment:
- `s/paasta-k8s-test-cluster/$KIND_CLUSTER/`
- break up the line after semicolon please, use backslash so that it's still the same shell
- you can drop `export` and semicolon to pass the env var directly